File: C++/bubblesort.cpp
//Author: D Vishal Reddy
//The following program is an Implementation of bubble sort.
//Time Complexity O(n^2)
#include<iostream>
using namespace std;
int main()
{
    int n,temp;
    cout << "Enter number of elements in the array\n";
    cin >> n;
    int arr[n];
    cout << "Enter the elements of the array\n";
    for(int i=0;i<n;i++)
    {
        cin >> arr[i];
    }
    for(int i=0;i<n;i++)
    {
        for(int j=i+1;j<n;j++)
        {
            if(arr[i]>arr[j])
            {
                temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    cout << "Sorted Array\n";
    for(int i=0;i<n;i++)
    {
        cout << arr[i] << " ";
    }
    cout << endl;
}

File: Day 01/Java/bubbleSort.java
public class BubbleSortinJava {  
    static void bubbleSort(int[] arr) {  
        
  
    }  
    public static void main(String[] args) {  
                int arr[] ={3,60,35,2,45,320,5};  
                int n = arr.length;  
                int temp = 0;  
                  
                for(int i=0; i < n; i++){  
                 for(int j=1; j < (n-i); j++){  
                     if(arr[j-1] > arr[j]){  
                         //swap elements  
                         temp = arr[j-1];  
                         arr[j-1] = arr[j];  
                         arr[j] = temp;  
                     }
                 }  
                }  
                 
                System.out.println("After Applying Bubble Sort to the Array");  
                for(int i=0; i < arr.length; i++){  
                        System.out.print(arr[i] + " ");  
                }  
   
        }  
}  
